5_databases/week2/part3.sql

```diff
@@ -0,0 +1,20 @@
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT *
+FROM task JOIN `user` on email LIKE "%@spotify.com"
```

Comment:
Here you should use the user_task many-to-many table to make relations so you get the task and not the user